### PR TITLE
Enable alternate registries to be able to be used.

### PR DESCRIPTION
**Pull request by kfox1111**
Fixes issue #5278

### CODE REVIEW

File: helm/vitess/templates/_pmm.tpl [hunk 1]
```diff
@@ -50,7 +51,7 @@ spec:
     spec:
       containers:
         - name: pmm
-          image: "percona/pmm-server:{{ $pmm.pmmTag }}"
+          image: "{{.Values.pmm.repo}}:{{ $pmm.pmmTag }}"
```
Comment (dkhenry) [inline helm/vitess/templates/_pmm.tpl]: The name of the image was removed here.

**Comment by morgo** [comment]
@kfox1111 ping

**Comment by sougou** [comment]
This PR has gone stale. @kfox1111 do you want to rework it? Otherwise, I'll close.

**Comment by kfox1111** [comment]
Sorry. I don't have time at the moment to refactor it. If we end up moving forward with vitess and the functionality isnt in by that time, I might take another stab at it.

**Pull request closed without merging**